packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -155,6 +154,195 @@ export const makeZoeStorageManager = (
+          );
+          state.instanceState.addIssuer(keyword, issuerRecord);
+          return issuerRecord;
+        },
+        makeZoeMint(
+          keyword,
+          assetKind = AssetKind.NAT,
+          displayInfo,
+          { elementShape = undefined } = {},
+        ) {
+          const { state, facets } = this;
+          // Local indicates one that zoe itself makes from vetted code,
+          // and so can be assumed correct and fresh by zoe.
+          const issuerBaggage = makeScalarBigMapStore('IssuerBaggage', {
+            durable: true,
+          });
+          const localIssuerKit = makeDurableIssuerKit(
+            issuerBaggage,
+            keyword,
+            assetKind,
+            displayInfo,
+            state.adminNode.terminateWithFailure,
+            { elementShape },
+          );
+          zoeMintBaggageSet.add(issuerBaggage);
+          return facets.helpers.wrapIssuerKitWithZoeMint(
+            keyword,
+            localIssuerKit,
+            state.adminNode,
+          );
+        },
+        registerFeeMint(keyword, allegedFeeMintAccess) {
+          const { state, facets } = this;
+          const feeIssuerKit = feeMint.getFeeIssuerKit(allegedFeeMintAccess);
+          return facets.helpers.wrapIssuerKitWithZoeMint(
+            keyword,
+            feeIssuerKit,
+            state.adminNode,
+          );
+        },
+        getInstanceRecord() {
+          const { state } = this;
+          return state.instanceState.getInstanceRecord();
+        },
+        getIssuerRecords() {
+          const { state } = this;
+          return issuerStorage.getIssuerRecords(
+            // the issuerStorage is a weakStore, so we cannot iterate over
+            // it directly. Additionally, we only want to export the
+            // issuers used in this contract instance specifically, not
+            // all issuers.
+            Object.values(
+              state.instanceState.getInstanceRecord().terms.issuers,
+            ),
+          );
+        },
+        initInstanceAdmin(instanceHandle, instanceAdmin) {
+          return instanceAdminStorage.updater.initInstanceAdmin(
+            instanceHandle,
+            instanceAdmin,
+          );
+        },
+        deleteInstanceAdmin(i) {
+          instanceAdminStorage.updater.deleteInstanceAdmin(i);
+        },
+        makeInvitation(
+          handle,
+          desc,
+          customProps = undefined,
+          proposalShape = undefined,
+        ) {
+          const { state } = this;
+
+          // If the contract-provided customProperties include the
+          // properties 'description', 'handle', 'instance',
+          // 'installation', their corresponding
+          // values will be overwritten with the actual values. For
+          // example, the value for `instance` will always be the actual
+          // instance for the contract, even if customProperties includes
+          // a property called `instance`.
```

Comment:
You changed the parameter name to `customProps` but the comment still talks about `customProperties`. In this case, I think I prefer the original longer name anyway.

Reply:
Btw, I like that you moved the `makeInvitation` implementation here. Now that I see it, the old extra level of indirection seems completely pointless.